server/publicapi/packages/service.py

```diff
@@ -9,13 +9,13 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 import logging
-from superdesk.services import BaseService
+from publicapi.items.service import ItemsService
 
 
 logger = logging.getLogger(__name__)
 
 
-class PackagesService(BaseService):
+class PackagesService(ItemsService):
```

Comment:
Oh OK, viewing the items/packages this way (i.e. as a combination of metadata and content) actually makes the hierarchy to feel more "natural".

@mugurrus Can we be sure that every potential contributor out there will have a good enough oversight of the project to be able to detect when refactoring is needed? :wink: (at least until we have good documentation in place).